tests/common/docker-compose-ssh-server.yaml

```diff
@@ -0,0 +1,23 @@
+version: "2.1"
+services:
+  scitacean-test-ssh-server:
+    image: lscr.io/linuxserver/openssh-server:latest
+    container_name: openssh-server
+    hostname: scitacean-test-ssh-server
```

Comment:
I guess you can add 
```
command: chown blahblah
```

Or creating the image with the entrypoint commands...?

```
FROM lscr.io/linuxserver/openssh-server:latest

RUN chmod blahblah
```